.github/workflows/build.yml

```
@@ -48,10 +52,25 @@ jobs:
           mkdir build
           cmake -S . -B build
           build-wrapper-linux-x86-64 --out-dir ${{ env.BUILD_WRAPPER_OUT_DIR }} cmake --build build/ --config Release
+      - name: Run tests to generate coverage information
+        run: |
+          ./build/coverage_test1
+          mv default.profraw test1.profraw
+          ./build/coverage_test2
+          mv default.profraw test2.profraw
+      - name: Merge counters from different runs and morf them into llvm-cov-compatible form
```

I'm not sure what "morf" means...
```suggestion
      - name: Merge counters from different runs and convert the result into an llvm-cov-compatible form
```